docs/assets/joining_parallel_gateway_doc.bpmn

```diff
@@ -0,0 +1,233 @@
+    <bpmn:task id="Activity_0np5087" name="Increment number of taken sequence flows">
+      <bpmn:incoming>Flow_0sekarg</bpmn:incoming>
+      <bpmn:outgoing>Flow_1avosat</bpmn:outgoing>
+    </bpmn:task>
+    <bpmn:startEvent id="Event_1igkqew" name="Take outgoing sequence flows">
+      <bpmn:outgoing>Flow_1ifhvyu</bpmn:outgoing>
+    </bpmn:startEvent>
+    <bpmn:task id="Activity_1nqfakp" name="ProcessInstance SequenceFlow Taken">
+      <bpmn:incoming>Flow_1ifhvyu</bpmn:incoming>
+      <bpmn:outgoing>Flow_0sekarg</bpmn:outgoing>
+    </bpmn:task>
+    <bpmn:sequenceFlow id="Flow_1ifhvyu" sourceRef="Event_1igkqew" targetRef="Activity_1nqfakp" />
+    <bpmn:sequenceFlow id="Flow_0sekarg" sourceRef="Activity_1nqfakp" targetRef="Activity_0np5087" />
+    <bpmn:task id="Activity_0kpi37x" name="ProcessInstance Activate Element">
+      <bpmn:incoming>Flow_1avosat</bpmn:incoming>
+      <bpmn:outgoing>Flow_00vq8qm</bpmn:outgoing>
+    </bpmn:task>
+    <bpmn:sequenceFlow id="Flow_1avosat" sourceRef="Activity_0np5087" targetRef="Activity_0kpi37x" />
+    <bpmn:exclusiveGateway id="Gateway_01jetkt" name="Parallel gateway can be activated">
+      <bpmn:incoming>Flow_00vq8qm</bpmn:incoming>
+      <bpmn:outgoing>Flow_0o85air</bpmn:outgoing>
+      <bpmn:outgoing>Flow_19jy9i0</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:sequenceFlow id="Flow_00vq8qm" sourceRef="Activity_0kpi37x" targetRef="Gateway_01jetkt" />
+    <bpmn:task id="Activity_0zca2sa" name="ProcessInstance Element Activating">
+      <bpmn:incoming>Flow_0o85air</bpmn:incoming>
+      <bpmn:outgoing>Flow_1luh5vk</bpmn:outgoing>
+    </bpmn:task>
+    <bpmn:sequenceFlow id="Flow_0o85air" name="yes" sourceRef="Gateway_01jetkt" targetRef="Activity_0zca2sa" />
+    <bpmn:task id="Activity_182ap6o" name="Activate command rejected">
+      <bpmn:incoming>Flow_19jy9i0</bpmn:incoming>
+      <bpmn:outgoing>Flow_1hzq222</bpmn:outgoing>
+    </bpmn:task>
+    <bpmn:sequenceFlow id="Flow_19jy9i0" name="no" sourceRef="Gateway_01jetkt" targetRef="Activity_182ap6o" />
+    <bpmn:endEvent id="Event_0mm8r9p">
+      <bpmn:incoming>Flow_1hzq222</bpmn:incoming>
+    </bpmn:endEvent>
+    <bpmn:sequenceFlow id="Flow_1hzq222" sourceRef="Activity_182ap6o" targetRef="Event_0mm8r9p" />
+    <bpmn:task id="Activity_1360769" name="ProcessInstance Element Activated">
+      <bpmn:incoming>Flow_1luh5vk</bpmn:incoming>
+      <bpmn:outgoing>Flow_1qay9is</bpmn:outgoing>
+    </bpmn:task>
+    <bpmn:sequenceFlow id="Flow_1luh5vk" sourceRef="Activity_0zca2sa" targetRef="Activity_1360769" />
+    <bpmn:endEvent id="Event_1u4nt5u">
+      <bpmn:incoming>Flow_1qay9is</bpmn:incoming>
+    </bpmn:endEvent>
+    <bpmn:sequenceFlow id="Flow_1qay9is" sourceRef="Activity_1360769" targetRef="Event_1u4nt5u" />
+    <bpmn:textAnnotation id="TextAnnotation_1m4mds7">
+      <bpmn:text>Note: this command does more than just what is modelled here. This model only focuses on the parallel gateway</bpmn:text>
```

Comment:
This note refers to an activity named `ProcessInstance SequenceFlowTaken` referring to it as a command, but this record is an event. Also note that commands and events don't do anything on their own. It's either that a command is processed, or an event is applied.

🔧 Please consider changing the wording.